README.md

```diff
@@ -8,11 +8,17 @@ Tested against:
 
 - CE 1.9
 - EE 1.14 (inc FPC)
- 
+
 # Usage
 
 Enable the extension in `System > Configuration > Advanced > Developer > JavaScript Settings > Move JavaScript to Footer`.
 
+Add block names with JS which must not be moved to footer in `System > Configuration > Advanced > Developer > JavaScript Settings > FooterJS Excluded Blocks`.
+Enter block names separated with Comma. Please note, JS libraries are moved to footer so be carefull with excluding JS which requires that libraries.
```

Comment:
Minor: Can we tidy up the grammar in this sentence please.  Perhaps:

"Enter block names separated with a comma. Please note, JS libraries are moved to footer so be careful excluding JS which has dependencies."